### PR TITLE
190 Simulator - e2e tests for running the tutorial

### PR DESCRIPTION
**Pull request by bradleySuira**

### CODE REVIEW

File: frontend/src/components/Tutorial/TutorialContainer.vue [hunk 1]
```diff
@@ -1,396 +1,287 @@
...
+      notify({
+        title: 'OK',
+        text: 'Contract method called',
+        type: 'success'
+      })
     }
   },
   {
-    target: '#tutorial-node-output',
+    target: () => '#tutorial-node-output',
```
Comment (cristiam86) [inline frontend/src/components/Tutorial/TutorialContainer.vue:66]: @bradleySuira why are these changed to functions?

File: frontend/test/e2e/Tutorial.spec.ts [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+    driver = await getDriver()
+    await driver.manage().setTimeouts({ implicit: 10000 })
+    contractsPage = new ContractsPage(driver)
+  })
+
+  it('Should show the welcome step and navigate to the next', async () => {
+    await contractsPage.navigate()
+    await contractsPage.waitUntilVisible()
+
+    const welcomeStepElement = await driver.wait(
```
Comment (cristiam86) [inline frontend/test/e2e/Tutorial.spec.ts:22]: @bradleySuira, could you please wrap these driver calls so they can be reused easily?

**Review by cristiam86 — CHANGES_REQUESTED** [review]
@bradleySuira can you please confirm if https://github.com/yeagerai/genlayer-simulator/pull/228 is already into this one?

File: frontend/src/components/Tutorial/TutorialContainer.vue (hunk 1 above)
Comment (bradleySuira) [reply frontend/src/components/Tutorial/TutorialContainer.vue]: fixed removed arrow funds

File: frontend/test/e2e/Tutorial.spec.ts (hunk 2 above)
Comment (bradleySuira) [reply frontend/test/e2e/Tutorial.spec.ts]: done, added page utils and utilize func to validate all steps

**Review by cristiam86 — APPROVED** [review]
